tests/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+project(Tests)
+
+include(${CMAKE_CURRENT_SOURCE_DIR}/../cmake/test_base.cmake)
+
+file(GLOB_RECURSE FOUND_SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/*.cpp")
```

Comment:
Globbing in cmake to search for sources is generally discouraged - the rationale is that during normal development, you're not expected to be rerunning cmake on each build (unless you explicitly change cmake files themselves) - so the glob is executed only once and won't reflect addition of new files.

Reply:
The ODR test may be an exception, though, I'll have to think about it.